$NetBSD: patch-Makefile.am,v 1.2 2018/11/03 12:51:54 leot Exp $

Do dynamic linking of libtool libraries too (otherwise bin/jq can
ends up to be statically linked).

--- Makefile.am.orig	2023-12-13 19:24:02.000000000 +0000
+++ Makefile.am
@@ -126,7 +126,7 @@ CLEANFILES = src/version.h .remake-versi
 
 bin_PROGRAMS = jq
 jq_SOURCES = src/main.c src/version.h
-jq_LDFLAGS = -static-libtool-libs
+jq_LDFLAGS = 
 jq_LDADD = libjq.la -lm
 
 if ENABLE_ALL_STATIC
